crates/revm/revm-inspectors/src/tracing/js/bindings.rs

```diff
@@ -229,12 +238,15 @@ impl MemoryObj {
                     }
                     let start = start as usize;
                     let end = end as usize;
+                    let size = end - start;
```

Comment:
maybe move conversions before condition

Reply:
hmm, the condition checks for `< 0` since js numbers are signed.

Reply:
ah it's fine, you also need it for formatting the error